src/gseproject/templates/game/scan.html

```diff
@@ -9,5 +9,201 @@
     The Scan function can be accessed from any of the game pages 
     If you are filling it in remember that ass the js functions in base_game_template.js are included, so DO NOT redefine those functions  
  -->
+<style>
+    .scan_icon_container {
+        position: absolute;
+        justify-content: center;
+        align-items: center;
+        top: 12%;
+        max-width: 300px;
+        max-height: 600px;
+    }
 
-{% endblock %}
+    .scan_icon_container img {
+        justify-content: center;
+        align-items: center;
+        max-width: 100%;
+        max-height: 100%;
+        transform: rotate(-10deg);
+        transition: transform 0.3s ease;
+        /* Add transition for smooth scaling */
+
+    }
+
+    .scan_icon_container img:hover {
+        transform: rotate(10deg);
+    }
+
+    .start_button_container {
+        position: absolute;
+        justify-content: center;
+        align-items: center;
+        top: 60%;
+        max-width: 200px;
+        max-height: 400px;
+    }
+
+    .start_button_container img {
+        justify-content: center;
+        align-items: center;
+        max-width: 100%;
+        max-height: 100%;
+        transition: transform 0.3s ease;
+        /* Add transition for smooth scaling */
+
+    }
+
+    .start_button_container img:hover {
+        transform: scale(0.9);
+        /* Scale the image down to 90% on hover */
+    }
+</style>
+{% load static %}
+<link rel="stylesheet" type="text/css" href=" {% static 'css/shop_styles.css' %}">
+
+<div class="image-container">
+
+    <div class="scan_icon_container">
+        <img src="{% static 'images/scan_icon.png' %}" alt="scan_icon" onclick="scanForCoordinate()">
+    </div>
+
+
+    <div class="start_button_container">
+        <img src="{% static 'images/START.png' %}" alt="start_button" onclick="scanForCoordinate()">
+    </div>
+</div>
+
+<script>
+
+
+
+
+    // Predefined array of buildings with names and coordinates
+    const buildings = [
```

Comment:
This needs to be a separate file

Reply:
This file is from marco. Please deny this and check his pull

Reply:
i cant deny individual files. plases claen up the whole request if you dont want it part of the request